src/types/model/participant.ts

```diff
@@ -0,0 +1,18 @@
+import {BaseElement} from './base/index';
+
+/**
+ * Describes a BPMN Participant of a collaboration.
```

Comment:
I guess if we talk about the BPMN specific collaboration we should write it with a capital letter.

```suggestion
 * Describes a BPMN Participant of a Collaboration.
```